<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DgcUi</class>
 <widget class="QDialog" name="DgcUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DgcApp</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>figuras_geometricas.gif</normaloff>figuras_geometricas.gif</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="layout">
     <item row="0" column="0">
      <layout class="QGridLayout" name="layoutLienzos">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QGridLayout" name="layAir"/>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="layPersp"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinAndMaxSize</enum>
             </property>
             <item>
              <widget class="QLabel" name="LabAir">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Aerea ( X, Z )</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignJustify|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="LabPersp">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>  Perspectiva</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QGridLayout" name="layFront"/>
           </item>
           <item>
            <widget class="Line" name="line_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="layLat"/>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="Line" name="line_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinAndMaxSize</enum>
             </property>
             <item>
              <widget class="QLabel" name="LabFront">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Frontal ( X, Y )</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="LabLat">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Lateral ( Z, Y )</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignJustify|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="layoutMenu">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="gbFig">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Figuras</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="0">
             <widget class="QPushButton" name="pbDibujar">
              <property name="text">
               <string>Dibujar</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>clip-art-pencil-eraser.png</normaloff>clip-art-pencil-eraser.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QComboBox" name="cbFigOpt">
              <property name="modelColumn">
               <number>0</number>
              </property>
              <item>
               <property name="text">
                <string>Ninguno</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Cubo</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Piramide</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Objeto de revolución</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Objeto de revolución - Perfil</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pbBorrar">
              <property name="text">
               <string>Borrar</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>Eraser-icon.png</normaloff>Eraser-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="Nmerid">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QPushButton" name="pbEjes">
              <property name="toolTip">
               <string comment="pepe" extracomment="hola"/>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string>Ejes</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>ejes_transformaciones.png</normaloff>ejes_transformaciones.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Meridianos</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="gbTrans">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Transformaciones</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QComboBox" name="cbTransform">
            <item>
             <property name="text">
              <string>Ninguna</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Trasladar</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rotar</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rotar sobre su eje</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Escalar</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pbObjColor">
            <property name="text">
             <string>Reflection Color</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>color.png</normaloff>color.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pbColor">
            <property name="text">
             <string>Object Color</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>color.png</normaloff>color.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lbObjColor">
            <property name="text">
             <string>Ref. Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lbColor">
            <property name="text">
             <string>Obj Color</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="gbCapas">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Capas</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>212</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::NoDragDrop</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerItem</enum>
            </property>
            <property name="layoutMode">
             <enum>QListView::SinglePass</enum>
            </property>
            <property name="viewMode">
             <enum>QListView::ListMode</enum>
            </property>
            <property name="modelColumn">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Ocultacion / Sombreado</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QComboBox" name="cbShading">
            <item>
             <property name="text">
              <string>Ninguno</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Z-Buffer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Difusa sin reflexión</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Difusa con reflexión</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="LightIntensity">
              <property name="text">
               <string>Intensidad</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="intensityVal"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="LightX">
              <property name="text">
               <string>Posición X </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LightXVal"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="LightY">
              <property name="text">
               <string>Posición Y </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LightYVal"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="LightZ">
              <property name="text">
               <string>Posición Z </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LightZVal"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
